src/app/components/Input/MenuButton.vue

```diff
@@ -0,0 +1,49 @@
+<template>
+  <button
+    class='menu-button'
+    :style="{
+      'font-weight':styles.includes('bold') ? 'bold' : 'normal', /* Applies text styling */
+      'font-style':styles.includes('italics') ? 'italic' : 'normal',
+      'text-decoration':styles.includes('underline') ? 'underline' : 'normal',
```

Comment:
I am not sure if it would make sense, but is there a clean way to merge the logic here with the `style` getter in MarkdownWrapButton? Maybe make a bigger utility function?